README.md

```diff
@@ -294,6 +294,80 @@ Executing run...
 	Your job took 420 seconds to complete successfully.
 ```
 
+#### Abort single or multiple jobs from CloudOS
+
+Aborts jobs in the CloudOS workspace that are either running or initialising.
+
+Usage: 
+```console
+cloudos job abort \
+    --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --workspace-id $WORKSPACE_ID \
+    --job-ids "ID1,ID2,ID3,ID4"
+```
+
+Example leading to an error (missing actual job ids):
+```console 
+cloudos job abort \
+    --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --workspace-id $WORKSPACE_ID \
+    --job-ids ""
+```
+Error message:
+```console
+Aborting jobs...
+[ERROR] No job IDs provided. Please specify at least one job ID to abort.
+```
+
+
+Example leading to warning (job id 012498120597125 does not exist):
+```console
+cloudos job abort \
+    --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --workspace-id $WORKSPACE_ID \
+    --job-ids "680a3cf80e56949775c02f16,012498120597125"
+```
+Warning message:
+```console
+Aborting jobs...
+[WARNING] Failed to get status for job 012498120597125: Server returned status 404. Reason: Not Found
+```
+
+Example of successful execution:
+```console
+cloudos job abort \
+    --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --workspace-id $WORKSPACE_ID \
+    --job-ids "680a3cf80e56949775c02f16"
+
+CloudOS python package: a package for interacting with CloudOS.
+
+Version: 2.20.0
+
+CloudOS job functionality: run and check jobs in CloudOS.
```

Comment:
```suggestion
```
I'd remove this part or else, we will need to update it every time we have a new version of cloudos-cli